python-practice-codes/given number is prime or not.py

```diff
@@ -0,0 +1,15 @@
+# Write a program in Python to check given number is prime or not.
+
+
+num = int(input("enter number: ")
+if num<=1:
+    print("{} is not prime number".format(num))
+else:
+    is_prime = True
+    for i in range(2,num//2+1):

```

Comment:
Please add one more solution to this with more optimized version of  this code using log function.

Reply:
done